simulate_scanning_strategy.py

```diff
@@ -65,9 +66,9 @@ def load_parameters(sim: lbs.Simulation) -> Parameters:
         output_nside=planet_params["output_nside"],
         output_map_file_name=sim.base_path / "map.fits.gz",
         output_table_file_name=sim.base_path / "observation_time_table.txt",
-        radius = scanning_params["radius"],
-        L2_orbital_velocity_rad_s = scanning_params["L2_orbital_velocity_rad_s"]
-        )
+        radius_au=scanning_params["radius_au"],
```

Comment:
Great! Just one small change: to make old TOML files work with this new version, better to provide a default value for these parameters:

```python
    radius_au=scanning_params.get("radius", 0.0),   # Using .get instead of [] permits to specify a default
    L2_orbital_velocity_rad_s=scanning_params.get("L2_orbital_velocity_rad_s", 0.0),
```

In this way, if a TOML file does not provide these parameters, the old behavior (i.e., keeping the spacecraft fixed in L2) is maintained.